src/Autoloader.php

```diff
@@ -190,17 +191,19 @@ public static function loadClass($class)
      */
     public static function findFile($class)
     {
-        if (isset(static::$classMap[$class]) && file_exists(static::$classMap[$class])) {
-            return static::$classMap[$class];
-        } else {
-            foreach (static::$psr4 as list($prefix, $path)) {
-                if (0 === strpos($class, $prefix)) {
-                    $classWithoutPrefix = substr($class, strlen($prefix));
-                    $file = $path.str_replace('\\', DIRECTORY_SEPARATOR, $classWithoutPrefix).'.php';
-
-                    if (file_exists($file)) {
-                        return $file;
-                    }
+        $lower = strtolower($class);
```

Comment:
Wow... it is very sad I did not know that :(

`$prefix` should be lowered in `addPsr4()` as well:
- `$prefix = trim($prefix, '\\').'\\';` => `$prefix = strtolower(trim($prefix, '\\')).'\\';`